scripts/start_local_env.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+# remember to chmod +x start_local_env.sh
+kotorekisteri_start_tmux() {
+  SESS_NAME=kotorekisteri
+  PANE1=kotorekisteri
+  REPO_ROOT=${1:-'.'}
+
+  (
+    cd $REPO_ROOT
+
+    # Install dependencies
+    echo "Installing dependencies..."
+    mise install
+
+    # Use old session if exists
+    tmux has-session -t $SESS_NAME 2>/dev/null
+    if [ $? -eq 0 ]; then
+      tmux attach -t $SESS_NAME
+      # TODO: Instead of return, kill the old session.
+      return
```

Comment:
Näin se poistais vanhan session, jos haluais, että scripti luo kaiken aina alusta.
```suggestion
tmux kill-session $SESS_NAME
```